### PR TITLE
Correction: fix link in user roles doc

### PR DESCRIPTION
**Pull request by matthijskrul**
## Summary of the changes / Why this is an improvement
- Fixes link in user roles doc

## Checklist

 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/user-roles.rst [hunk 1]
```diff
@@ -29,7 +29,7 @@ CrateDB Cloud user roles
 
 This section describes the roles that can be set for users within CrateDB
 Cloud. For information on how to do so, see our documentation on ref:`adding
-users <crate-reference:add-users>`.
+users <crate:administration_user_management>`.
```
Comment (amotl) [inline docs/user-roles.rst:32]: According to [1], the prefix is really actually `crate-reference`?

[1] https://github.com/crate/crate-docs-theme/blob/0.15.3/src/crate/theme/rtd/conf/__init__.py#L75

Comment (amotl) [reply docs/user-roles.rst:32]: How would the tests succeed in this case?

Comment (matthijskrul) [reply docs/user-roles.rst:32]: If you look at the actual document as it is now, you'll see the link is not a functional link: https://crate.io/docs/cloud/reference/en/latest/user-roles.html#cratedb-cloud-user-roles

As far as I can tell the actual repo is crate/docs and the relevant Sphinx header is administration_user_management?

Comment (matthijskrul) [reply docs/user-roles.rst:32]: Oh it looks like the problem is the :ref: link is lacking a colon 😳

**Comment by matthijskrul** [comment]
@amotl now it should be correct!

**Review by amotl — APPROVED** [review]
Thank you.

However, I still wonder how the tests defined at https://github.com/crate/cloud-reference/blob/master/.github/workflows/docs.yml could have succeeded before? [1] shows no problems beforehand, after improving the documentation infrastructure.

Is this something we should investigate? Would `make check` invoked on your machine croak without this patch? That should actually the expectation, otherwise our test harness is moot.

[1] https://github.com/crate/cloud-reference/actions

**Comment by matthijskrul** [comment]
I think because it never did :ref: properly, it was never parsed as a link and therefore the link checker did not check whether it was a valid one?

**Comment by amotl** [comment]
> I think because it never did `:ref:` properly?

Ah, _now_ I am seeing it. Thanks!

**Pull request merged**